### PR TITLE
Link to organizations page

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # OpenTermsArchive.org
 
-This is the repository for `opentermsarchive.org`, the website that presents the [Open Terms Archive](https://github.com/OpenTermsArchive/engine) project.
+This is the repository for `opentermsarchive.org`, the website that presents the [Open Terms Archive](https://github.com/OpenTermsArchive/) project.
```
Comment (MattiSG) [inline README.md:3]:
```suggestion
The website that presents Open Terms Archive to the general public.
```

**Review by MattiSG — COMMENTED** [review]
What about [line 164](https://github.com/OpenTermsArchive/opentermsarchive.org/pull/214/files#diff-9b9077ac1888c7edbfc04f673de566963e4b89b7b313e8c27bcc5481eb5ae13aR164)?

**Comment by clementbiron** [comment]
> What about [line 164](https://github.com/OpenTermsArchive/opentermsarchive.org/pull/214/files#diff-9b9077ac1888c7edbfc04f673de566963e4b89b7b313e8c27bcc5481eb5ae13aR164)?

Thanks you for your careful review!

**Review by MattiSG — APPROVED** [review]
👏

**Pull request merged**